src/Symfony/Bundle/SecurityBundle/Resources/config/security_authenticator.php

```diff
@@ -163,4 +167,42 @@
             ])
             ->tag('monolog.logger', ['channel' => 'security'])
     ;
+
+    if (class_exists(AbstractAccessTokenAuthenticator::class)) {
```

Comment:
Can we bump the security-http requirement to `^6.2` instead? That would prevent leftovers such as #46438

Reply:
Yes sure. I was not 100% sure this was what you expect. I was not award of the issue.

Reply:
No worries, it was a real question. I'm not sure why we decided to favor such class existence checks before, that's not what we're used to do. Maybe Wouter or someone else can bring some light

Reply:
The tests with `--prefer-lowest` are green now.
I don't understand the one failing with PHP8.2. There is no error description; just `Process completed with exit code 1.`

Reply:
It's unrelated to your changes, please ignore it (unless you really want to investigate that failure in another PR of course)

Reply:
I'm not sure what I did with login throttling, but I general I think we avoid version-related checks for symfony packages (except for x.4 & (x+1).0 releases).

Btw, we started to split out authenticator services to their own DI config file (`security_authenticator_login_link.php` & `security_authenticator_remember_me.php`). We might want to do the same for access token.

And at last: class existence checks should always be avoided in the DI config files, but moved to the DI extension instead.

Reply:
I split and put the definitions in a dedicated file as suggested.
There is no class existence check anymore.